### DISCUSSION TITLE
是否支持能像Lottie一樣對單一層進行染色?

### DISCUSSION THREAD

**Discussion by aaron-juan** [Tencent/libpag#2531] [Q&A]
如Android，在LottieAnimationView能夠對單一層進行染色

`addValueCallback(`
`    KeyPath("animations"),`
`    LottieProperty.COLOR_FILTER,`
`    { PorterDuffColorFilter(color, PorterDuff.Mode.SRC_ATOP) }`
`)`

**Comment by kevingpqi123** [comment]
类似于 BlendMode 的效果吗，这类功能目前 pag 没有暴露接口给外部修改，设计师可以在 AE 中对图层添加此类效果，接入层暂时没办法修改
